### ISSUE TITLE
Failed to train well using the preset parameters 

### ISSUE THREAD

**Issue by Owen-Liuyuxuan** [georgesung/deep_rl_acrobot#2]
Tensorflow 1.10
Ubuntu 16.04
Python 2.7

First, thank you for your contribution. Here is the issue.

I first changed tf.mul into tf.multiply to make tensorflow compatible to my current version.
Then I tried both "python learn_agent.py" and "python full_training.py" but none of them succeed in producing a non-trivial final model.

I would like to know if there is anything wrong or maybe this is just not my lucky day to do random initialization?

I have recently been learning RL and try to learn to make a non-trivial model in acrobot. So thank you if you are here to help.

![screenshot from 2018-11-08 11-37-36](https://user-images.githubusercontent.com/31618608/48176387-bd6e2f00-e34a-11e8-89f9-a3ead8f77732.png)

**Comment by luisriera** [comment]
Hi Owen,

I have noted that on `full_training.py`, there are two phases.  On the first phase, the program runs for 1500 episode, then on the second phase the learning rate is reduced by 10 and runs for 8500 episodes.  Have you changed any of this parameters? If not, the training is going to take some time.
